pkgs/tools/X11/xdg-utils/default.nix

```diff
@@ -2,7 +2,7 @@
 # docs deps
 , libxslt, docbook_xml_dtd_412, docbook_xml_dtd_43, docbook_xsl, xmlto
 # runtime deps
-, resholve, bash, coreutils, dbus, file, gawk, glib, gnugrep, gnused, jq, nettools, procmail, procps, xdg-user-dirs
+, resholve, bash, coreutils, dbus, file, gawk, glib, gnugrep, gnused, jq, lockfileProgs, nettools, procmail, procps, which, xdg-user-dirs
```

Comment:
```suggestion
, resholve, bash, coreutils, dbus, file, gawk, glib, gnugrep, gnused, jq, nettools, procmail, procps, which, xdg-user-dirs
```

Reply:
Thanksf for that catch! I think that crept in in the merge. BTW I've also removed `fetchpatch` from the inputs as that function is not currently used. Can always be added back in later if needed. Hope everything is correct now. Many thanks!